Docs/source/glossary.rst

```diff
@@ -5,4 +5,34 @@ Glossary
 .. glossary::
 
```

Comment:
We have a more extensive glossary that I pasted into the google doc underneath this one.

Reply:
Go for it! The glossary should ideally not be used for abbreviations, which should be in-line where it makes sense, but I think with the complexity of this documentation having a page that defines all this is not a bad idea. 

I don't think it's a great idea to have this in the Glossary, so I'd separate it into Glossary and a separate page for Abbreviations